mscore/preferences.h

```diff
@@ -27,6 +27,8 @@
 
 namespace Ms {
 
+extern QString mscoreGlobalShare;
```

Comment:
I remember the compiler complain about not finding that.  Let me test again without that line and get back to you...